pandas/computation/__init__.py

```diff
@@ -7,22 +7,13 @@
 try:
     import numexpr as ne
     ver = ne.__version__
-    _NUMEXPR_INSTALLED = ver >= LooseVersion('2.1')
+    _NUMEXPR_INSTALLED = ver >= LooseVersion('2.4.6')
 
-    # we specifically disallow 2.4.4 as
-    # has some hard-to-diagnose bugs
-    if ver == LooseVersion('2.4.4'):
-        _NUMEXPR_INSTALLED = False
-        warnings.warn(
```

Comment:
Yep.  Addressed in https://github.com/pandas-dev/pandas/pull/15383/commits/f22559876b130891e7c834cf269d1289307e3fa3.